MyThreadPool/MyThreadPool/MyThreadPool.cs

```diff
@@ -0,0 +1,289 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace MyThreadPool;
+
+/// <summary>
+/// A thread pool that manages a fixed number of threads to execute submitted tasks.
+/// </summary>
+public class MyThreadPool
+{
+    private readonly Thread[] _threads;
+    private readonly Queue<Action> _taskQueue = new Queue<Action>();
```

Comment:
```suggestion
    private readonly Queue<Action> _taskQueue = new();
```
Не пишите один тип дважды, тут и ниже